### PR TITLE
docs: add chau tran to GDE resources

### PR DESCRIPTION
**Pull request by nartc**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -164,6 +164,14 @@
     "picture": "cexbrayat.jpg",
     "bio": "Author of `Become a ninja with Angular (2+)` https://books.ninja-squad.com/angular - Angular trainer and @Ninja-Squad co-founder"
   },
+  "chautran": {
```
Comment (petebacondarwin) [inline aio/content/marketing/contributors.json:167]: Please can you change the key to your github username, which I believe is [`nartc`](https://github.com/nartc)?

**Review by twerske — APPROVED** [review]
LGTM & welcome!

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Thanks @nartc - please can you also reduce the image to less than 30K?

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (nartc) [reply aio/content/marketing/contributors.json]: Updated

**Review by petebacondarwin — APPROVED** [review]

**Comment by petebacondarwin** [comment]
Eek!

> Error: The following keys in 'contributors.json' are not in alphabetical order: 'nartc' and 'chrisnoring'.

**Comment by nartc** [comment]
@petebacondarwin 🤦‍♂️ I just pushed to put my section in the right order.

**Comment by petebacondarwin** [comment]
One last fix:

> CircleCI config on master has been modified since commit 73105aa, which this PR is based on.
>
> Please rebase the PR on master after fetching from upstream.

**Comment by petebacondarwin** [comment]
Who'd have thought this was going to be so difficult, eh?

> Error: The following keys in 'contributors.json' are not in alphabetical order: 'nartc' and 'mmalerba'.

**Comment by petebacondarwin** [comment]
I've fixed the sort-order issue and force pushed.

**Comment by nartc** [comment]
@petebacondarwin oh didn't see your comment. I was outside and did another force push before checking. My bad.

**Comment by petebacondarwin** [comment]
> @petebacondarwin oh didn't see your comment. I was outside and did another force push before checking. My bad.

🙀 you didn't `--force-with-lease`??

(no problem, I was just trying to make things easier for you)

**Comment by nartc** [comment]
@petebacondarwin ouch, I must have messed up my alias there. Is there anything I need to do?

**Pull request closed without merging**